doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -209,6 +209,236 @@ The `mz_sessions` table contains a row for each active session in the system.
+in the system on user objects to user roles.
+
+<!-- RELATION_SPEC mz_internal.mz_show_object_privileges -->
+| Field            | Type     | Meaning                                         |
+|------------------|----------|-------------------------------------------------|
+| `grantor`        | [`text`] | The role that granted the privilege.            |
+| `grantee`        | [`text`] | The role that the privilege was granted to.     |
+| `database`       | [`text`] | The name of the database containing the object. |
+| `schema`         | [`text`] | The name of the schema containing the object.   |
+| `name`           | [`text`] | The name of the object.                         |
+| `object_type`    | [`text`] | The type of object the privilege is granted on. |
+| `privilege_type` | [`text`] | They type of privilege granted.                 |
+
+### `mz_show_role_members`
+
+The `mz_show_role_members` view contains a row for each role membership in the system.
+
+<!-- RELATION_SPEC mz_internal.mz_show_role_members -->
+| Field     | Type     | Meaning                                                 |
+|-----------|----------|---------------------------------------------------------|
+| `role`    | [`text`] | The role that `member` is a member of.                  |
+| `member`  | [`text`] | The role that is a member of `role`.                    |
+| `grantor` | [`text`] | The role that granted membership of `member` to `role`. |
+
+### `mz_show_schema_privileges`
+
+The `mz_show_schema_privileges` view contains a row for each schema privilege granted
+in the system on user schemas to user roles.
+
+<!-- RELATION_SPEC mz_internal.mz_show_schema_privileges -->
+| Field            | Type     | Meaning                                         |
+|------------------|----------|-------------------------------------------------|
+| `grantor`        | [`text`] | The role that granted the privilege.            |
+| `grantee`        | [`text`] | The role that the privilege was granted to.     |
+| `database`       | [`text`] | The name of the database containing the schema. |
+| `name`           | [`text`] | The name of the schema.                         |
+| `privilege_type` | [`text`] | They type of privilege granted.                 |
+
+### `mz_show_system_privileges`
+
+The `mz_show_system_privileges` view contains a row for each system privilege granted
+in the system on to user roles.
+
+<!-- RELATION_SPEC mz_internal.mz_show_system_privileges -->
+| Field            | Type     | Meaning                                     |
+|------------------|----------|---------------------------------------------|
+| `grantor`        | [`text`] | The role that granted the privilege.        |
+| `grantee`        | [`text`] | The role that the privilege was granted to. |
+| `privilege_type` | [`text`] | They type of privilege granted.             |
+
+### `mz_show_my_all_privileges`
```

Comment:
nit: maybe `mz_show_all_my_privileges` sounds more logical, even if it's not technically consistent with e.g. `mz_show_my_cluster_privileges`?

Reply:
Yeah, I was going back and forth on this. I'll switch it to `mz_show_all_my_privileges`.